### PR TITLE
Beta urls

### PR DESCRIPTION
**Pull request by ktsamis**
## What does this PR change?

Correct res and sle11 beta urls for mirror from BETA to Beta. 
Also fix detection of release in RES 7 case for tools_pool_repo

### CODE REVIEW

**Comment by ktsamis** [comment]
@moio I put you as reviewer because I'm not sure about the commit about detection 937c1a7 it's been there for a while and I'm surprised we didn't see it before. Am I wrong in fixing it? Maybe it is correct?

File: salt/repos/default.sls [hunk 1]
```diff
@@ -415,9 +415,9 @@ tools_pool_repo:
     {% else %}
     - baseurl: http://{{ grains.get("mirror") | default("download.suse.de/ibs", true) }}/SUSE/Products/RES/{{ release }}-CLIENT-TOOLS/x86_64/product/
     {% endif %}
-    {% elif grains.get('mirror') %}
+    {% elif release == 7 %}
```
Comment (juliogonzalez) [inline salt/repos/default.sls:418]: Well... don't really understand why this change is here, but if you tested it, I guess it's OK

File: salt/repos/default.sls [hunk 2]
```diff
@@ -436,7 +436,7 @@ tools_update_repo:
   pkgrepo.managed:
     - humanname: tools_update_repo
     {% if 'beta' in grains.get('product_version') | default('', true) %}
-    - baseurl: http://{{ grains.get("mirror") | default("download.suse.de/ibs", true) }}/SUSE/Updates/RES/{{ release }}-CLIENT-TOOLS-BETA/x86_64/update/
+    - baseurl: http://{{ grains.get("mirror") | default("download.suse.de/ibs", true) }}/SUSE/Updates/RES/{{ release }}-CLIENT-TOOLS-Beta/x86_64/update/
```
Comment (juliogonzalez) [inline salt/repos/default.sls:439]: This is NOT Ok.

Check https://download.suse.de/ibs/SUSE/Updates/RES/ and you will see all beta client tools are `N-CLIENT-TOOLS-BETA` Could it be you have problems at your mirror?

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
Are you sure the changes from `BETA`  to `Beta` are needed in all cases? At least the last change is clearly wrong.

**Review by juliogonzalez — APPROVED** [review]
LGTM now.

**Comment by srbarrios** [comment]
@juliogonzalez @ktsamis I think I already commented about these lines in the past. I think we should homogenize the way to handle the mirror in our repos. Instead of using this if statement that is hard to read, I would use the format we are using in other states by replace the domain directly inside the repo url.

**Pull request merged**